webapp/src/components/Controls/SwitchToggler.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { FormGroup, FormControlLabel, Switch } from "@mui/material";
+
+type LabelPlacement = "top" | "bottom" | "start" | "end";
+
+type Props = {
+  label: string;
+  labelPlacement: LabelPlacement;
+  enable?: true;
+  onChange: (toggleValue: boolean) => void;
+};
+
+const SwitchToggler: React.FC<Props> = ({
+  label,
+  labelPlacement,
+  enable,
+  onChange,
+}) => {
+  return (
+    <FormGroup aria-label="position" row>
+      <FormControlLabel
+        control={
+          <Switch
+            checked={enable ?? false}
+            onChange={(_, checked) => onChange(checked)}
+          />
+        }
+        label={label}
+        labelPlacement={labelPlacement ?? undefined}
```

Comment:
The ` ?? undefined` won't do anything.
```suggestion
        labelPlacement={labelPlacement}
```